shell/shared/admin.js

```diff
@@ -16,10 +16,10 @@
 
 const ADMIN_TOKEN_EXPIRATION_TIME = 15 * 60 * 1000;
 const publicAdminSettings = [
-  "google", "github", "emailToken", "splashUrl", "signupDialog",
+  "google", "github", "ldap", "emailToken", "splashUrl", "signupDialog",
   "adminAlert", "adminAlertTime", "adminAlertUrl", "termsUrl",
   "privacyUrl", "appMarketUrl", "appIndexUrl", "appUpdatesEnabled",
-  "serverTitle", "returnAddress",
+  "serverTitle", "returnAddress", "ldapNameField",
```

Comment:
I find it kinda weird that the client-side code has to know about this to fill in the profile defaults, rather than it being something the server adapts into well-known locations, but I guess that can happen client-side and probably nothing will break